soc/arm/nordic_nrf/nrf52/Kconfig.soc

```diff
@@ -64,6 +64,7 @@ config SOC_NRF52811
 	select HAS_HW_NRF_PWM0
 	select HAS_HW_NRF_QDEC
 	select HAS_HW_NRF_RADIO_IEEE802154
+	select HAS_HW_NRF_RADIO_BLE_CODED
```

Comment:
Keep the alphabetical order, please. In `Kconfig.peripheral` as well.

Reply:
Right. Will do that, thanks.

Reply:
Fixed